### ISSUE TITLE
Example

### ISSUE THREAD

**Issue by CHEWX** [pristas-peter/wp-graphql-gutenberg-acf#2]
Hi,

Do you have an example of querying Advanced Custom Fields?

Thanks,
Tom

**Comment by pristas-peter** [comment]
Hi, here i this, however there is an issue with acf pro, that needs to be sorted out.
https://github.com/AdvancedCustomFields/acf/issues/192

```graphql
{
  pages(where: {name: "index"}) {
    edges {
      node {
        id
        blocks {
          ...on AcfBannerCarouselBlock {
            acf {
              items {
                image {
                  status
                }
                link {
                  title
                }
              }
            }
          }
        }
      }
    }
  }
}

```

**Comment by CHEWX** [comment]
Ok, be good to get that into a README.

So I assume everything will return blank/null currently? As this is what I am getting?

**Comment by pristas-peter** [comment]
I guess. Try to put multiple acf blocks into the content and if you edit only one of them, the other should work. I have a sort of fix for this, but want to fix it on the acf pro side, not mine.

**Comment by pristas-peter** [comment]
Fixed in #3
